app/views/LocationTracking.js

```diff
@@ -25,8 +25,11 @@ import BackgroundImageAtRisk from './../assets/images/backgroundAtRisk.png';
 import Colors from '../constants/colors';
 import LocationServices from '../services/LocationService';
 //import BroadcastingServices from '../services/BroadcastingService';
+import BackgroundTaskServices, {
+  executeTask,
```

Comment:
See the other note about this.  I think `check_intersect` could be in `Intersect.js`, and used here directly.  Would help with readability for sure.